full-service/src/service/sync.rs

```diff
@@ -136,7 +145,24 @@ pub fn sync_all_accounts(
 
             continue;
         }
-        sync_account_next_chunk(ledger_db, conn, &account.id, logger)?;
+        let (num_received_txos, num_spent_txos) =
```

Comment:
Do we want to fire the webhook on spending? I was imagining only firing the webhook on deposits (new txos).

practically speaking, there may be little difference as RTH proscribes sending a 0 value change txp whenever a txo is spent.

Reply:
great point - will just do received